### DISCUSSION TITLE
Orthographic camera

### DISCUSSION THREAD

**Discussion by SineVector241** [Mojang/minecraft-editor#152] [Feedback]
It would be nice to have an orthographic camera in editor mode that we can snap between the x, y and z axis. This is extremely useful in any 3D editor e.g. Cad/Modelling Software. Mostly would be useful in world painting with top down view but again. Would be really nice to have for accurate scaling and placement.

**Comment by Wabbabrick** [comment]
I strongly support this feature request; it would significantly improve the workflow for map makers.

Additionally, could we please consider adding Numpad Keyboard Shortcuts for this feature? For example, using the Numpad keys for quick rotation (e.g., 4/6 to rotate left/right) or for switching between modes would make this tool much faster to use. This would align the Bedrock Editor closer to industry-standard 3D tools like Blender, where the Numpad is essential for efficient navigation and manipulation.

**Reply by m-brisch** [reply]
Great idea, I added this into the triage queue.

In the meantime, if you click on the compass in the upper-right corner it'll snap to an axis but I agree having a keyboard shortcut would be super useful.
